ext/mbstring/mbstring.stub.php

```diff
@@ -65,29 +65,29 @@ function mb_list_encodings(): array {}
 
 function mb_encoding_aliases(string $encoding): array {}
 
-function mb_encode_mimeheader(string $string, ?string $charset = null, ?string $transfer = null, string $linefeed = "\r\n", int $indent = 0): string {}
+function mb_encode_mimeheader(string $string, ?string $charset = null, ?string $transfer_encoding = null, string $linefeed = "\r\n", int $indent = 0): string {}
 
 function mb_decode_mimeheader(string $string): string {}
 
-function mb_convert_kana(string $string, string $option = "KV", ?string $encoding = null): string {}
+function mb_convert_kana(string $string, string $mode = "KV", ?string $encoding = null): string {}
 
-function mb_convert_variables(string $to, array|string $from, mixed &$var, mixed &...$vars): string|false {}
+function mb_convert_variables(string $to_encoding, array|string $from_encoding, mixed &$var, mixed &...$vars): string|false {}
 
-function mb_encode_numericentity(string $string, array $convmap, ?string $encoding = null, bool $is_hex = false): string {}
+function mb_encode_numericentity(string $string, array $map, ?string $encoding = null, bool $use_hex = false): string {}
 
-function mb_decode_numericentity(string $string, array $convmap, ?string $encoding = null): string {}
+function mb_decode_numericentity(string $string, array $map, ?string $encoding = null): string {}
 
 function mb_send_mail(string $to, string $subject, string $message, array|string $additional_headers = [], ?string $additional_parameters = null): bool {}
```

Comment:
WDYT?
```suggestion
function mb_send_mail(string $to, string $subject, string $message, array|string $additional_headers = [], ?string $additional_params = null): bool {}
```
Although, as I wrote in the PDO rename PR, we could/should set up some rules for using `$param` and `$arg`.

Reply:
Done.